SimpleWAWS/SimpleWAWS.csproj

```diff
@@ -26,6 +26,7 @@
     <UseGlobalApplicationHostFile />
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
+    <ApplicationInsightsResourceId>/subscriptions/9fa73032-4951-4cdc-a734-d0910ccdca7f/resourcegroups/StandardResourceGroup/providers/microsoft.insights/components/SimpleWAWS</ApplicationInsightsResourceId>
```

Comment:
let's remove this as well from here. This is only needed for VS to connect to the AI instance.